CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+2.8.0 (2023-05-28)
+------------------
```

Comment:
headers are inserted at release time... I should add a comment to this file to clarify that.